lib/html_parser.dart

```diff
@@ -64,7 +64,15 @@ class HtmlParser extends StatelessWidget {
       cleanedTree,
     );
 
-    return StyledText(textSpan: parsedTree, style: cleanedTree.style);
+    // This is the final scaling that assumes any other StyledText instances are
+    // using textScaleFactor = 1.0 (which is the default). This ensures the correct
+    // scaling is used, but relies on https://github.com/flutter/flutter/pull/59711
+    // to wrap everything when larger accessibility fonts are used.
+    return StyledText(
+      textSpan: parsedTree, 
+      style: cleanedTree.style,
+      textScaleFactor: MediaQuery.of(context).textScaleFactor,
```

Comment:
This should be added as a parameter, It's still breaking accessibility functions. When textScaleFactor != 1.0

Reply:
I have a branch with a working solution. This does everything you did plus, what's not working for me. Don't have permission to push it tho xD.

Reply:
I agree that being able to pass in `textScaleFactor` would be a useful enhancement. I'll push up a change containing that today.

When you say that it is "still breaking accessibility functions", I'm not sure what you mean. What is broken?

Reply:
As screenshots are showing when I am allowing the user to define `textScaleFactor ` via accessibility functions on his phone (I've made font smaller, not bigger). Everything was shifted to the left and smaller (mainly width). (this will be later fixed in https://github.com/flutter/flutter/pull/59711) I've modified my local flutter code to verify.

Reply:
ok. I see now... so, I imagine the width problem will be resolved with the Flutter PR you mentioned... I'm just trying to work out if you're saying there needs to be additional changes beyond a) the changes in this PR and b) the Flutter PR.

Reply:
Any updates on this? I just tested this branch and the content no longer overflows the screen for me, but still overflows its container to the right. (On both flutter channels beta and master).